### PR TITLE
ER-1250 flow to recruit

### PR DESCRIPTION
**Pull request by shomavg**

### CODE REVIEW

File: src/Employer/Employer.Web/Configuration/TempDataKeys.cs [hunk 1]
```diff
@@ -8,5 +8,6 @@ public static class TempDataKeys
         public const string Qualifications = "Qualifications";
         public const string ApplicationReviewStatusInfoMessage = "ApplicationReviewStatus_InfoMessage";
         public const string VacancyClosedMessage = "VacancyClosedMessage";
+        public static string EmployerVacancyMessage = "EmployerVacancy_InfoMessage";
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Configuration/TempDataKeys.cs]:
```suggestion
        public const string CreateVacancyShortcutIndicator = "CreateVacancyShortcutIndicator";
```

File: src/Employer/Employer.Web/Controllers/Part1/EmployerVacancyController.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        public IActionResult CreateVacancy()
+        {
+            TempData.Remove(TempDataKeys.EmployerVacancyMessage);
+            return RedirectToRoute(RouteNames.CreateVacancy_Get);
+        }
+
+        [HttpGet("manage-vacancy", Name = RouteNames.EmployerManageVacancy_Get)]
+        public IActionResult ManageVacancy()
+        {
+            TempData.Add(TempDataKeys.EmployerVacancyMessage, string.Format(InfoMessages.VacancyUpdated, "fromMAHome"));
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Controllers/Part1/EmployerVacancyController.cs]:
```suggestion
            TempData.Add(TempDataKeys.CreateVacancyShortcutIndicator, 1);
```

File: src/Employer/Employer.Web/Views/CreateVacancyOptions/Options.cshtml [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ﻿@model Esfa.Recruit.Employer.Web.ViewModels.CreateVacancyOptions.CreateVacancyOptionsViewModel
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        
+        <esfaFeatureDisabled name="@FeatureNames.AllowLevyPayingEmployersOnly">
+            <a asp-show="@Model.HasClonableVacancies" asp-route="@RouteNames.Dashboard_Index_Get" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Return to your vacancies</a>
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Views/CreateVacancyOptions/Options.cshtml]: Shouldn't this be going to `RouteNames.Dashboard_Account_Home`

Comment (shomavg) [reply src/Employer/Employer.Web/Views/CreateVacancyOptions/Options.cshtml]: Return to your vacancies, in the prototype, takes flow back to the employer dashboard page.

File: src/Employer/Employer.Web/Controllers/EmployerVacancyController.cs [hunk 4]
```diff
@@ -9,6 +10,7 @@ public class EmployerVacancyController : Controller
         [HttpGet("employer-create-vacancy", Name = RouteNames.EmployerCreateVacancy_Get)]        
         public IActionResult CreateVacancy()
         {
+            TempData[TempDataKeys.ReferredFromMAHome] = "Return to home";
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Controllers/EmployerVacancyController.cs]: This is weird. Why should this controller be setting the content of a link on another page?  This should be more of a boolean flag and then the page inspecting it then decides how to handle this information.

File: src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs [hunk 5]
```diff
@@ -37,11 +38,25 @@ public async Task<TitleViewModel> GetTitleViewModel(string employerAccountId)
             var vm = new TitleViewModel
             {
                 PageInfo = new PartOnePageInfoViewModel(),
-                ShowReturnToMALink = dashboard == null || !dashboard.CloneableVacancies.Any()
             };
+            GetBackLink(dashboard, vm);
             return vm;
         }
 
+        private static void GetBackLink(EmployerDashboard dashboard, TitleViewModel vm)
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs]: This is named incorrectly. Its setting not getting.

Can we get rid of this.  Can we put back the `ShowReturnToMALink` and just have 2 links on the view. One with a `asp-show` the other with `asp-hide`. That way a FE dev can change the link text if needed.

**Pull request closed without merging**